browser/main/NoteList/index.js

```diff
@@ -758,14 +758,16 @@ class NoteList extends React.Component {
     return copy(noteLink)
   }
 
+  // Navigate seems unused
```

Comment:
It is executed when a global shortcut event is dispatched. See `componentDidMount`.

Reply:
ah, OK, thanks. I missed that.
I'll remove the comment and also the commented line on line 763.